deploy/deploy-sota-local.yml

```diff
@@ -78,9 +78,7 @@ services:
     ports:
       - 80:9000
     env:
-      CORE_PORT: "8080"
-      CORE_HOST: "core"
-      RESOLVER_PORT: "8081"
-      RESOLVER_HOST: "resolver"
+      CORE_API_URI: "http://core:8080"
+      RESOLVER_API_URI: "http://resolver:8081"
```

Comment:
I'd be inclined to leave this as separate configurable hosts and ports. It gives us more flexibility when deploying to different machines.